tests/test_init.py

```diff
@@ -68,13 +72,100 @@ def test_valid_raw_input_does_not_fail():
     remove_fake_cred_file()
 
 def test_board_can_be_instantiated_with_an_installable_and_sub_id():
-    apiKey = 'fakeApiKey'
-    apiSecret = 'fakeApiSecret'
-    subId = 'fakeSubId'
-
-    i = vestaboard.Installable(apiKey=apiKey, apiSecret=apiSecret, getSubscription=False, saveCredentials=False)
-    vb = vestaboard.Board(i, subscriptionId=subId)
+    i = vestaboard.Installable(apiKey=TEST_API_KEY, apiSecret=TEST_API_SECRET, getSubscription=False, saveCredentials=False)
+    vb = vestaboard.Board(i, subscriptionId=TEST_SUB_ID)
     vb.post('Should not error')
+def test_odd_length_center_pad():
+    small_board = return_valid_small_board()
+    i = vestaboard.Installable(apiKey=TEST_API_KEY, apiSecret=TEST_API_SECRET, getSubscription=False, saveCredentials=False)
+    vb = vestaboard.Board(i, subscriptionId=TEST_SUB_ID)
+    vb.raw(small_board, pad='center')
+    expected = [
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 5, 0, 0, 6, 0, 0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0],
+        [1, 2, 3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4, 0, 0, 1, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+    ]
+    assert small_board == expected
```

Comment:
Let's add the blank padding lines here between tests for consistency - some of the lower ones have them, but not all.